server/channels/store/sqlstore/system_store.go

```diff
@@ -16,14 +16,25 @@ import (
 
 type SqlSystemStore struct {
 	*SqlStore
+
+	systemInsertQuery sq.InsertBuilder
```

Comment:
You are over-complicating this. Just create a function that returns an array of columns names and call that function instead of using select *, See function `bookmarkWithFileInfoSliceColumns for reference.